### DISCUSSION TITLE
LibreSBC as voice service provider SBC in front of Registrar?

### DISCUSSION THREAD

**Discussion by hb9eue** [hnimminh/libresbc#169] [General]
Hi Minh

I work for a voice service provider serving residential customer and businesses.

We look for an SBC solution to ideally place in from of our registrar servers so have the UAs of our customer register through the SBC to our registrar instance.

We would favour an open source solution over a commercial solution as we have been disappointed by the lack of flexibility and inability to fix obvious bugs in commercial solutions we used in the past.

There are not many good SBC B2BUA open source solutions. You project sounds promising. We have installed LibreSBC on a machine in our lab and managed to do a very basic configuration, but we don't understand how to configure it as a transparent registrar SBC and are now not sure it is at all possible.

So, I guess, best is: We ask the development team.

To give you an overview what we try to achieve, I have attached a PDF with our actual situation and requirements.

[IMP SBC Requirements.pdf](https://github.com/user-attachments/files/16816498/IMP.SBC.Requirements.pdf)

I look forward in hearing from you, if you consider it possible to achieve our goals with libreSBC. If so, I would also like to ask if you, or maybe some partner which use your product, would offer some paid support to point us in the right direction to set up libreSBC and get us going.

-Benoît-

**Comment by hnimminh** [comment]
Hi @hb9eue, 
LibreSBC currently does not support transparent REGISTER. It's component was designed as an REGISTRAR to offload REGISTRAR request for core.
I have no ideal how you handle opensips/kamailio (basically they are the same) and why they can not fulfill your requirement, Since they are the best opensource for SIP proxy.
